### PR TITLE
UI rework

### PR DESCRIPTION
**Pull request by StijnOostdam**

### CODE REVIEW

File: Winleafs.Models/Models/Effects/Effect.cs [hunk 1]
```diff
@@ -40,5 +40,22 @@ public override string ToString()
...
+        public override bool Equals(object obj)
+        {
+            var effect = obj as Effect;
+
+            if (effect == null)
+            {
+                return false;
+            }
+
+            return ((Effect)obj).Name == Name;
```
Comment (bartdebever) [inline Winleafs.Models/Models/Effects/Effect.cs]: use effect here

File: Winleafs.Models/Models/Effects/Effect.cs [hunk 2]
```diff
@@ -40,5 +40,22 @@ public override string ToString()
         {
             return Name;
         }
+
+        public override bool Equals(object obj)
+        {
+            var effect = obj as Effect;
```
Comment (bartdebever) [inline Winleafs.Models/Models/Effects/Effect.cs]: Can also be written as 
```cs
if (obj is Effect effect)
{
    return effect.Name == Name;
}

return false;
```

File: Winleafs.Models/Models/UserSettings.cs [hunk 3]
```diff
@@ -390,6 +411,15 @@ private static JToken Migration_6_7(JToken jToken)
 
             return jToken;
         }
+
+
+        [Migration("7", "8")]
+        private static JToken Migration_7_8(JToken jToken)
+        {
+            jToken[nameof(Schedules)] = new JArray();
```
Comment (bartdebever) [inline Winleafs.Models/Models/UserSettings.cs]: Do keep in mind the bug that we had before, this may occur again

Comment (StijnOostdam) [reply Winleafs.Models/Models/UserSettings.cs]: Yes, noted.

**Pull request merged**